lib/permutations.rb

```diff
@@ -1,4 +1,21 @@
-
 def permutations?(string1, string2)
```

Comment:
This would fail for something like `heelo` and `hello`.  Instead of making `lookup_hash[letter] = true` instead count the number of times each letter appears.